lib/client.js

```diff
@@ -7,7 +7,7 @@ const { HTTPParser } = require('http-parser-js')
 const { EventEmitter } = require('events')
 const Request = require('./request')
 const assert = require('assert')
-const { Readable } = require('stream')
+const { PassThrough } = require('stream')
```

Comment:
I think this will slow down the normal case, can you check?

Reply:
It does. About 500 req/s less.

Reply:
I think we should have a slightly complicated approach that keeps using the readable with the previous API.

Reply:
I will fix this. Shouldn’t be too bad.

Reply:
PTAL. I've changed it so that you can return either a `Writable` or a `Readable`.